openvasd/openvasd.c

```diff
@@ -67,11 +44,11 @@ struct openvasd_connector
   gchar *host;    /**< server hostname. */
   gchar *scan_id; /**< Scan ID. */
   int port;       /**< server port. */
-  openvasd_vt_stream_t stream_resp; /** For response */
+  curlutils_response_stream_t stream_resp; /** For response */
 };
 
 /**
- * @brief Struct holding options for Openvasd parameters.
+ * @brief Struct holding options for openvasd parameters.
```

Comment:
I'd like to encourage you to do this kind of thing in a separate commit.

In general it's a whole lot easier to follow PRs that are broken down into simple clean steps, with a commit for each step.